### PR TITLE
fix: Integrate Crowdin to support translations

### PR DESCRIPTION
**Pull request by ErikQQY**
Fix: #11 

Signed-off-by: ErikQQY <2283984853@qq.com>

### CODE REVIEW

**Comment by ErikQQY** [comment]
@hsluoyz Plz review.

File: crowdin.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+project_id: '458576'
+api_token: '94f023f6ed164a9989930454897f55c11c166065d2f072b46adec92462ea35474f4a4f4b84fa9253'
```
Comment (hsluoyz) [inline crowdin.yml:2]: Is it safe to expose api_token to public? Can you see how Casbin site is doing this?

**Review by hsluoyz — CHANGES_REQUESTED** [review]

File: crowdin.yml (hunk 1 above)
Comment (ErikQQY) [reply crowdin.yml:2]: I will add the api to the secrets.

**Comment by ErikQQY** [comment]
I think it's better now

**Review by hsluoyz — APPROVED** [review]

**Pull request merged**